sentry-core/src/main/java/io/sentry/core/Breadcrumb.java

```diff
@@ -43,6 +43,22 @@
     this.timestamp = timestamp;
   }
 
+  /**
+   * Creates HTTP breadcrumb.
+   *
+   * @param url - the request URL
+   * @param method - the request method
+   * @return the breadcrumb
+   */
+  public static @NotNull Breadcrumb http(final @NotNull String url, final @NotNull String method) {
```

Comment:
we could add status code as well?

https://develop.sentry.dev/sdk/event-payloads/breadcrumbs/#http

wondering if the category should be `xhr`

Reply:
I assumed `xhr` term makes sense only in the context or browser interacting with a server.

Status is optional - this breadcrumb is added when request processing starts. On the go I think we could add more methods here.